app/kubernetes/components/datatables/applications-datatable-details/applications-datatable-details.controller.js

```diff
@@ -0,0 +1,12 @@
+import { KubernetesConfigurationTypes } from 'Kubernetes/models/configuration/models';
+
+export default class {
+  $onInit() {
+    const secrets = (this.configurations || [])
```

Comment:
why don't we get the secrets from the parent component?

Reply:
`item.Configurations` is passed down to this component.
The `item` object does not have `Secrets`; these need to be extracted out of the `Configurations` - which is what this component is doing.

Do let me know if there is an easier way to extract the secrets though.